src/View.Uno/Themes/Generic.xaml

```diff
@@ -1,6 +0,0 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Should this file really be removed?

Reply:
Make sure you test your changes using both the UWP and WinUI packages to validate that default styles are available for consumer applications.